### PR TITLE
go.mod vendor: ensure we never have the toolchain directive set

### PR DESCRIPTION
**Pull request by Luap99**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by baude** [comment]
lgtm

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module github.com/containers/podman/v5
 
-// Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates
+// Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates.
+// That generally means there should be no toolchain directive present.
```
Comment (edsantiago) [inline go.mod:4]: This wording confuses me a little: I might spend myself looking for a `toolchain` line. Would it make sense to rephrase as something closer to
> // Warning: if there is a "toolchain" directive anywhere in this file (and most of the time there shouldn't be), its version must be an exact match to the "go" directive

**Review by edsantiago — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (Luap99) [reply go.mod:4]: sure

**Comment by Luap99** [comment]
Good to merge?

**Comment by edsantiago** [comment]
/lgtm

was on my todo list for today, sorry for delay

**Pull request merged**